Java/History/Evolution of Java/README.md

```diff
@@ -1 +1,39 @@
+| J2SE 1.4     | 2002 | This release contained several important upgrades, enhancements, and additions. It added the new keyword assert, chained expections and a channel based I/O subsystem. It enhanced the collections framework and the networking classes.|
+| J2SE 5       | 2004 | J2SE 5 fundamentally expanded the scope, power and range of the language. The major new features included- generics, annotations, autoboxing and auto-unboxing, enumerations, enhanced for each style for loop, variable length arguments, static imports, formatted I/O, concurrency utilities. These functionalities added an entirely new dimension to java programming.|
+| Java SE 6 | 2006 | While Java SE 6 did not add any new features, it provided enhancement to API libraries, added several new features and offered improvements to the runtime.|
+| Java SE 7 | 2011 | It addressed several small features such as, a string could now be used in a switch statement, introducing binary integer literals, underscores in numeric literals, an expanded try statement, type reference when constructing a generic instance, enhanced exception handling  and better type checking.|
+| Java SE 8 | 2014 | It added an all new GUI library JavaFx along with a far-reaching feature, lambda expressions. It also included Date and Time API.|
+| Java SE 9 | 2017 | This update added java platform module system update, jshell, XML Catalog, and JLink. It removed JavaDB from the JDK.|
+| Java SE 10 | 2018 | It added features like local variable type interface, application class data sharing, garbage collector interface etc.|
+| Java SE 11 | 2018 | This update added multiple features including dynamic class file loader, HTTP client and Transport layer security. While several modules- JavaFx, Java EE and COBRA were- removed from JDK.|
+| Java SE 12 | 2019 | It added features like microbenchmark suite, JVM constant API, One AArch64Port, Default CDS Archives etc.|
+| Java SE 13 | 2019 | It resolved thousands of bugs and provided numerous smaller enhancements. It added dynamic CDS archives, reimplemented the Legacy Socket API and added preview features such as Switch Expressions and Text Blocks.|
+| Java SE 14 | 2020 | This update gave a stable functionality to Switch Expression which was in preview. It also added new preview features such as records and a second preview of Text Blocks.|
+| Java SE 15 | 2020 | It added support for Text Blocks and ZGC. Preview features such as Sealed classes, Hidden classes and pattern matching for ```instanceof```.  |
+| Java SE 16 | 2021 | JDK 16 removes Ahead-of-Time compilation and Graal JIT options. Complete support for record (which was a preview in earlier versions) was added along with pattern matching for ```instanceof```. |
+| Java SE 17 (LTS) | 2021 | JDK 17 is the current long term support (LTS). It provides support for Sealed classes and adds Pattern matching for switch (Preview).|
```

Comment:
Kindly mention the strict floating point change. It is major change actually, since from this, the `strictfp` modifier is not required. A lot of codes had to changed because of this now :no_mouth: